dev/AppNotifications/AppNotificationBuilder/AppNotificationButton.h

```diff
@@ -73,6 +73,8 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         winrt::hstring m_inputId{};
         bool m_useContextMenuPlacement{};
         AppNotificationButtonStyle m_buttonStyle { AppNotificationButtonStyle::Default };
+
+        NotificationTelemetryHelper m_telemetryHelper{};
```

Comment:
Can we not use the telemetry helper class from NotificationBuilder rather than declare and reinit again?